src/routes/(infoPages)/land/fire/safety/fire-danger-map/+page.svelte

```diff
@@ -92,7 +98,27 @@
             parishes,
             Plot.pointer(
               Plot.geoCentroid({
-                title: (d) => d.properties.parishname,
+                title: (d) => {
+                  const { parishname: parishName, fireRisk, ObservationDate } = d.properties;
+                  if (fireRisk === "0" || ObservationDate === "") {
+                    return `Parish: ${parishName}\nNo fire danger data available`;
+                  } else {
+                    let formattedDate;
+                    try {
+                      formattedDate = format(
+                        new Date(d.properties.ObservationDate),
+                        "MMMM d, yyyy",
+                      );
+
+                      return `Parish: ${parishName}\nFire Risk: ${
+                        fireRiskLevels[Number(fireRisk)]
```

Comment:
I think we have an off-by-one error here; I'm seeing the following on the preview:
* green: medium
* yellow: very high
* orange: extreme
* red: undefined

Reply:
ah, yup. forgot about the array index offset. easy fix.